config/environments/development.rb

```diff
@@ -77,4 +77,7 @@
 
   # Uncomment if you wish to allow Action Cable access from any origin.
   # config.action_cable.disable_request_forgery_protection = true
+  config.hosts << ".ngrok-free.app"
+
+  config.assets.digest = false
```

Comment:
Is there a reason why this is getting added?

Reply:
deleted. I think this was added before when I was trying to configure devise to work with jwt..